RallyHereIntegration/Source/RallyHereIntegration/Private/RH_LocalPlayerSessionSubsystem.cpp

```diff
@@ -792,8 +815,17 @@ void URH_LocalPlayerSessionSubsystem::OnPlatformSessionInviteAccepted(const bool
 
 	if (bSuccesful)
 	{
-		// we need to join the session
-		URH_PlatformSessionSyncer::JoinRHSessionByPlatformSession(this, Session, URH_OnlineSession::GetJoinDetailDefaults(this), FRH_GenericSuccessWithErrorBlock());
+		auto AuthContext = GetAuthContext();
+		if (!AuthContext.IsValid() || !AuthContext->IsLoggedIn())
```

Comment:
I wonder if it would be better to put this repeated code block in a inlined function or something

Reply:
If the code was larger, maybe.  But these are also two different (if very similar) use cases.  The activity case generally has more metadata, and often no session id.  The invite case always has a session id, but very little metadata.  So keeping them distinct in case we need to differentiate them later does not hurt us much.